me.broken_by.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,38 @@
+app-id: me.broken_by.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - "/app/bin/pmedit-cli"
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=home
+  - --persist=.java
```

Comment:
with the broad `home` permission you don't need persist.

Reply:
removed